cases-SEP24/sep10.test.js

```diff
@@ -315,14 +325,14 @@ describe("SEP10", () => {
     it("succeeds for a signer without an account", async () => {
       const kp = StellarSDK.Keypair.random();
       const { token, logs } = await getSep10Token(url, kp, [kp]);
-      expect(token).toBeTruthy();
+      expect(token).signersAssertion(logs, [kp], accountPool);
     });
 
     it("fails if a challenge for a nonexistent account has extra client signatures", async () => {
       const account = getAccount();
       const kp = StellarSDK.Keypair.random();
       const { token, logs } = await getSep10Token(url, kp, [kp, account.kp]);
-      expect(token, logs).toBeFalsy();
+      expect(token).not.signersAssertion(logs, [account.kp, kp], accountPool);
```

Comment:
Whats the difference between passing `logs` in `expect()` vs `signersAssertion()`?

Reply:
When I was looking into how to approach this logging problem and found out about `expect.extend(matchers)`
https://jestjs.io/docs/en/expect#expectextendmatchers
I couldn't find a solution to pass both token and logs in the expect fucntion's clause (`expect(token, logs)`) when I do. it the `logs` arg and all other args are shifted `orElseLogKeypairs(token, logs, accountsUsed, accountPool)`
token ✅
logs ❌ (has the accountsUsed value)
accountsUsed ❌ (has the accountPool value)
accountPool ❌ (null)

Reply:
I made the decision to not increase the time spent on finding a work around when I wrote a working solution.
Even if it's a bit awkward to write.
However I am open to change suggestions.